PM updates and new TS files (#72)
mulesoft/docs-partner-manager: modules/ROOT/pages/ts-deploy-not-in-effect.adoc

> = I recently updated and deployed my message flow, however, the configurations haven’t taken effect on the runtime B2B apps. Did I do this incorrectly?

Reviewer: ```suggestion
= I recently updated and deployed my message flow. However, the configurations haven’t taken effect on the runtime B2B apps. Did I do this incorrectly?
```

Author: F=Fixed.

ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> As you continue to use Anypoint Partner Manager, you may find the need to update the configuration associated with a previously deployed message flow.

Reviewer: ```suggestion
As you continue to use Anypoint Partner Manager, you might need to update the configuration associated with a previously deployed message flow.
```

Author: f

After updating the configuration, you still find your message flow executing with the previous or older set of configurations.

== Causes

Any of the following can lead to this type of error:

> * You may have not deployed the message flow after making changes to the message flow’s configuration.

Reviewer: ```suggestion
* You might have not deployed the message flow after making changes to the message flow’s configuration.
```

Author: f

> * You may be within the five minute synchronization window for changes from Anypoint Partner Manager to reflect on your B2B integration applications.

Reviewer: ```suggestion
* You might be within the five-minute synchronization window for changes from Anypoint Partner Manager to reflect on your B2B integration applications.
```

Author: f

== Solutions

> 

Reviewer: Consider a transition here between subtopic levels.

=== Deploy the Changed Message Flow

> Once you make changes to the message flow configuration, you need to deploy it in order for the new configurations to become effective on your B2B integration applications.

Reviewer: ```suggestion
After you change the message flow configuration, you must deploy it for the new configurations to become effective on your B2B integration applications.
```

Author: f

To deploy your message flow after making changes:

> . From within your Anypoint Partner Manager environment (either Sandbox or Production) select your <Hostname> profile.

Reviewer: ```suggestion
. From within your Anypoint Partner Manager environment (either Sandbox or Production), select your <Hostname> profile.
```

Author: f

. Select *Message Flows*.
. Search for your message flow by name.

> . Once you locate the message flow, click on the message flow card to open it up.

Reviewer: ```suggestion
. After you locate the message flow, click on the message flow card to open it.
```

Author: f

> . Confirm that each sections in the interface shows a green checkmark next to it in the flow, and click Deploy to deploy the configurations to Mule.

Reviewer: ```suggestion
. Confirm that each section in the flow shown in the interface displays a green checkmark next to it, and click Deploy to deploy the configurations to Mule.
```

Author: f

. Provide the necessary reason to trigger a new deployment, and wait for the process to complete.

> When the deployment is successfully completed, Anypoint partner manager responds with a successful notification.

Reviewer: ```suggestion
When the deployment is successfully completed, Anypoint Partner Manager responds with a success notification.
```

> === Wait for the Synchronization Window to Complete

Reviewer: ```suggestion
=== Wait for the Synchronization Window to Finish
```
Except consider rewording, because "a window" does not "finish."
(Note that "complete" requires an object; look it up in the style guide.)

Author: f

> Your runtime B2B integration applications executing on your Mule runtimes and receives the latest configuration from the Anypoint Partner Manager control plane at a five minute interval.

Reviewer: Please reread for accuracy; the existing version has syntax errors:
```suggestion
Your runtime B2B integration applications executing on your Mule runtimes receives the latest configuration from the Anypoint Partner Manager control plane at five-minute intervals.
```

Author: f

> Depending on when you made the configuration changes to your message flows in Partner Manager and deployed them,

Reviewer: Ditto:
```suggestion
Depending on when you made the configuration changes to your message flows and deployed them, it
```

Author: f

> It can take a maximum interval of five minutes for your most recent changes to take effect on the runtime B2B integration applications.

Reviewer: ```suggestion
can take up to five minutes for those changes to take effect on the runtime B2B integration applications.
```

Author: f
